Overview.src.html

```diff
@@ -3979,9 +3975,52 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 is itself associated with <span title=concept-Response-response>response</span>'s
 <span title=concept-response-header-list>header list</span>.
 
-<p>A <code>Response</code> object's <span title=concept-Body-body>body</span> is its
-<span title=concept-Response-response>response</span>'s
-<span title=concept-response-body>body</span>.
+<p>A <code>Response</code> object also has an associated
+<dfn title=concept-Response-readable-stream>readable stream</dfn> of type
+<span title=concept-ReadableStream>ReadableStream</span> which is initially null.
+
+<p class = "note no-backref">The type of the associated
+<span title=concept-Response-readable-stream>readable stream</span> is defined as
+<span title=concept-ReadableStream>ReadableStream</span>, but it is currently being discussed.
+Depending on the discussion, the type might be changed to <code>ReadableByteStream</code>. Hence the
+type name is not strictly specified at this moment, though the behavior is specified because
+<code>ReadableByteStream</code> has all methods and properties which
+<span title=concept-ReadableStream>ReadableStream</span> has.
```

Comment:
I think it would be clearer to have something like `<p class=XXX><a href={linktoissue}>This might become a <code>ReadableByteStream</code></a>.`